### DISCUSSION TITLE
Add words segmentation plugin?

### DISCUSSION THREAD

**Discussion by yale8848** [meilisearch/product#240] [Feedback & Feature Proposal]
Word segmentation is most important in search engine. So can meilisearch add words segmentation by user customize ，may be is a plugin?

**Comment by gmourier** [comment]
Hi @yale8848 👋

I'm not sure I understand. When you use the term word segmentation in your previous post, are you referring to tokenization?

If so, the need would be to be able to define separators/behaviors to better segment words based on a language?

Thanks for your answer!
